tricycle_controller/include/tricycle_controller/tricycle_controller.hpp

```diff
@@ -106,8 +106,8 @@ class TricycleController : public controller_interface::ControllerInterface
     const std::string & traction_joint_name, std::vector<TractionHandle> & joint);
   CallbackReturn get_steering(
     const std::string & steering_joint_name, std::vector<SteeringHandle> & joint);
-  double convert_trans_rot_vel_to_steering_angle(double v, double omega, double wheelbase);
-  std::tuple<double, double> twist_to_ackermann(double linear_command, double angular_command);
+  //double convert_trans_rot_vel_to_steering_angle(double v, double omega, double wheelbase);
```

Comment:
please don't leave commented code in PRs, we use version control software to be able to recover old code